doc/source/scihub.rst

```diff
@@ -26,19 +25,18 @@ and see the help message of the script.
 
 Some usage examples are:
 
-
   - Download via the doi number:
 
     .. code:: bash
 
-      papis scihub 10.1002/andp.19053220607 \\
+      papis scihub 10.1002/andp.19053220607 \
```

Comment:
It looked to me like the double backtick is redundant (and actually shows in the doc as double backtick). Please confirm this is the case and doesn't break any example scripts.

Reply:
`papis scihub` is a bit dead itself, so I'm not too worried about breaking anything. Agree that it should be just a single backslash though!

Reply:
Oh, good to know.

There were a few other example scripts with double backslashes too. But yes I believe a single backslash is what was intended.